src/modules/vtol_att_control/standard.cpp

```diff
@@ -309,12 +309,16 @@ void Standard::fill_actuator_outputs()
 
 	/* fixed wing controls */
 	_actuators_out_1->timestamp = _actuators_fw_in->timestamp;
-	_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] = -_actuators_fw_in->control[actuator_controls_s::INDEX_ROLL]
-			* (1 - _mc_roll_weight);	//roll
+
+	_actuators_out_1->control[actuator_controls_s::INDEX_ROLL] =
+		(-_actuators_fw_in->control[actuator_controls_s::INDEX_ROLL] + _params->fw_roll_trim);
```

Comment:
I feel this is treating symptoms and we should still try to understand underlying causes before adding more parameters.

Are you sure you don't just have a fixed-wing tuning issue? If adding a constant trim value here is able to level it out that would suggest that means the controller should be able to do it itself if correctly tuned.

If it is only during the initial transition period that you lack roll/pitch authority have you tried increasing the blending threshold so it doesn't start reducing MC input until a higher airspeed is reached? If you feel MC control isn't powerful enough to correct during transition (is your setup still underpowered?) perhaps we need to address how blending works (ie: nonlinear; full FW control surface when transition starts; decouple MC/FW actuator blending).

Also wrt pitch is your mechanical setup such that zero pitch in MC mode is also zero pitch for FW?

This change also results in the control surfaces always being active in MC mode (needs testing in wind). So on one hand the control surfaces will now still move while in MC mode and there will also be offsets (trims) present which are intended to offset motor torque that isn't present in MC mode. Holding position in a headwind in MC mode with off center ailerons for example would result in inducing roll.